types/recordrtc/index.d.ts

```diff
@@ -228,7 +228,9 @@ declare class RecordRTC {
     resumeRecording(): void;
 
     /** auto stop recording after specific duration */
-    setRecordingDuration(): void;
+    setRecordingDuration(): {
+        onRecordingStopped: (callback: (...args: any[]) => any) => any;
+    };
```

Comment:
based on [the source code](https://github.com/muaz-khan/RecordRTC/blob/eb16816757121bcfcbd198d394d4d47459cadd22/dev/RecordRTC.js#L405-L414), I think we can type this a bit stricter using two overloads:

```suggestion
    setRecordingDuration(milliSeconds: number): {
        onRecordingStopped: (callback: () => void) => void;
    };
    /** auto stop recording after specific duration */
    setRecordingDuration(milliSeconds: number, onRecordingStopped: () => void): void;
```

I would also suggest you copy-paste [these examples](https://github.com/muaz-khan/RecordRTC/blob/eb16816757121bcfcbd198d394d4d47459cadd22/dev/RecordRTC.js#L405-L414) into the test file to ensure it passes.